### DISCUSSION TITLE
make slider velocity not affect distance spacing

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#18197] [Ideas] [answered]
for a real life example of why this is useful is that i want to have a consistent spacing for every time i have a slider end on an 1/8th note and a note immediately after it; i want a set spacing every time. the perfect spacing is 0.5x for me, but i notice as i change slider velocity, this changes accordingly. it would be nice to have 1.8x spacing correspond to 1.8x velocity, but have them independent too

**Comment by peppy** [comment]
This goes against the whole premise of what distance spacing is. At 1.0x it is coupled to slider velocity. This is required to make... beatmapping work. Logically.

**Reply by Axilotl17** [reply]
okay, what do you recommend to get consistent spacing of a pattern repeatedly then?

**Reply by Walavouchey** [reply]
people usually either eyeball it, or do something like this:

https://user-images.githubusercontent.com/36758269/167992833-73fdd308-abb9-48ac-8c4d-f56f85eb236a.mp4

**Comment by Walavouchey** [comment, marked as answer]
This might have come about from confusion about what distance spacing is, since it's a tool to help maintain time-distance equality, not visual spacing. The 'x' in 1.0x probably isn't enough to indicate to a new user at first glance that it's a multiplier to the distance you get when moving at the same velocity as a slider.

I'm not entirely sure if https://github.com/ppy/osu/pull/18117 solves this ~~(I for one have now idea why there are multiple rings when you can adjust DS to achieve the same thing)~~, but maybe the setting needs an explanatory tooltip.

**Reply by peppy** [reply]
> I'm not entirely sure if #18117 solves this (I for one have now idea why there are multiple rings when you can adjust DS to achieve the same thing), but maybe the setting needs an explanatory tooltip.

Have you actually tried it? Because what you are saying is actually not correct. The multiple rings do not represent different distance spacing values, but different points in time.

**Reply by Walavouchey** [reply]
I tried it out while reporting https://github.com/ppy/osu/discussions/18229 and yeah apparently that detail didn't register in my brain

**Reply by peppy** [reply]
I was hoping the highlighting of the circles helps as time progresses (added in the last release).
